packages/cypress/src/generators/cypress-project/cypress-project.spec.ts

```diff
@@ -239,12 +239,45 @@ describe('schematic:cypress-project', () => {
             linter: Linter.EsLint,
           });
           const packageJson = readJson(tree, 'package.json');
-          const eslintrcJson = readJson(tree, 'apps/my-app-e2e/.eslintrc.json');
-
           expect(
             packageJson.devDependencies['eslint-plugin-cypress']
           ).toBeTruthy();
-          expect(eslintrcJson.extends).toContain('plugin:cypress/recommended');
+
+          const eslintrcJson = readJson(tree, 'apps/my-app-e2e/.eslintrc.json');
+          expect(eslintrcJson).toMatchInlineSnapshot(`
```

Comment:
This diff is purely the change to make the snapshot to capture the whole config, instead of just `extends`